cloudofficeprint/src/main/java/com/cloudofficeprint/RenderElements/Distribute.java

```diff
@@ -0,0 +1,53 @@
+package com.cloudofficeprint.RenderElements;
```

Comment:
this class is not necessary, move it to horizontal tabular loop as extra property.